pkg/jobrunaggregator/jobruntestcaseanalyzer/cmd.go

```diff
@@ -0,0 +1,322 @@
+	fs.StringVar(&f.Platform, "platform", f.Platform, "The platform used to narrow down a subset of the jobs to analyze, ex: aws|gcp|azure|vsphere")
+	fs.StringVar(&f.Infrastructure, "infrastructure", f.Infrastructure, "The infrastructure used to narrow down a subset of the jobs to analyze, ex: upi|ipi")
+	fs.StringVar(&f.Network, "network", f.Network, "The network used to narrow down a subset of the jobs to analyze, ex: sdn|ovn")
+	fs.IntVar(&f.MinimumSuccessfulTestCount, "minimum-successful-count", defaultMinimumSuccessfulTestCount, "minimum number of successful test counts among jobs meeting criteria")
+	usage := fmt.Sprintf("mutually exclusive to --payload-tag.  Matches the .label[%s] on the prowjob, which is a UID", jobrunaggregatorlib.PayloadInvocationIDLabel)
+	fs.StringVar(&f.PayloadInvocationID, "payload-invocation-id", f.PayloadInvocationID, usage)
+
+	fs.StringVar(&f.WorkingDir, "working-dir", f.WorkingDir, "The directory to store caches, output, and the like.")
+	fs.DurationVar(&f.Timeout, "timeout", f.Timeout, "Time to wait for analyzing job to complete.")
+	fs.Var(&jobGCSPrefixSlice{&f.JobGCSPrefixes}, "explicit-gcs-prefixes", "a list of gcs prefixes for jobs created for payload. Only used by per PR payload promotion jobs. The format is comma-separated elements, each consisting of job name and gcs prefix separated by =, like openshift-machine-config-operator=3028-ci-4.11-e2e-aws-ovn-upgrade~logs/openshift-machine-config-operator-3028-ci-4.11-e2e-aws-ovn-upgrade")
+}
+
+func NewJobRunsTestCaseAnalyzerCommand() *cobra.Command {
+	f := NewJobRunsTestCaseAnalyzerFlags()
+
+	/* Example runs for release controller:
+	 ./job-run-aggregator analyze-test-case
+	    --google-service-account-credential-file=credential.json
+	    --test-group=install
+	    --platform=aws
+	    --network=sdn
+	    --infrastructure=ipi
+	    --payload-tag=4.11.0-0.nightly-2022-04-28-102605
+	    --job-start-time=2022-04-28T10:28:48Z
+	    --minimum-successful-count=10
+
+	Example runs for PR based paylaod:
+	 ./job-run-aggregator analyze-test-case
+	    --google-service-account-credential-file=credential.json
+	    --test-group=install
+	    --payload-invocation-id=09406e30ea661e228c17120f28eff3c6
+	    --job-start-time=2022-03-18T13:10:20Z
+	    --minimum-successful-count=10
+	    --explicit-gcs-prefixes=periodic-ci-openshift-release-master-ci-4.11-e2e-aws-ovn-upgrade=logs/openshift-machine-config-operator-3028-ci-4.11-e2e-aws-ovn-upgrade,periodic-ci-openshift-release-master-ci-4.11-upgrade-from-stable-4.10-e2e-aws-ovn-upgrade=/logs/openshift-machine-config-operator-3028-ci-4.11-upgrade-from-stable-4.10-e2e-aws-ovn-upgrade
+	*/
+	cmd := &cobra.Command{
+		Use:          "analyze-test-case",
+		Long:         `Analyze status of a test case of certain group to make sure they meet minimum criteria specified`,
```

Comment:
I'd love to see this expanded to fully explain the feature. 
 
"Group" of what?
Where is it used?
Elaborate on the different uses between payload and PR payload.
Describe an example of how it's used given our current install test checks.
Why does this command exist? What value is it providing? How could someone use it in the future for more?